src/WebHelper.js

```diff
@@ -155,13 +155,14 @@ class WebHelper extends Helper {
             }
             return nets(Object.assign({
                 body: data,
-                method: method
+                method: method,
+                encoding: undefined // eslint-disable-line no-undefined
```

Comment:
We always want this because while the data we send using `store` varies in format, the response is always (expected to be) JSON. Is the right way to think about this?

Reply:
Right! Also if not, we can set `responseType` to `arraybuffer` when setting up a store that needs arraybuffer.